modules/core/include/opencv2/core/hal/intrin_riscv_vector.hpp

```diff
@@ -0,0 +1,490 @@
+
+#ifndef OPENCV_HAL_INTRIN_RVV_VEC_HPP
```

Comment:
maybe we should choose another name, e.g. intrin_rvv1.0.hpp or something like that?